docs/design/seccomp.md

```diff
@@ -202,11 +202,11 @@ use annotations instead of extending the API with new fields.
 In the alpha version of this feature we will use annotations to store the
 names of seccomp profiles.  The keys will be:
 
-`security.alpha.kubernetes.io/seccomp/container/<container name>`
+`container.seccomp.security.alpha.kubernetes.io/<container name>`
 
 which will be used to set the seccomp profile of a container, and:
 
-`security.alpha.kubernetes.io/seccomp/pod`
+`seccomp.security.alpha.kubernetes.io/pod`
```

Comment:
I think what @sttts has proposed is probably what we have to do if only a single slash is allowed.  I would rather not do something like `/container-foo` and parse it.